### ISSUE TITLE
intervalToHalfOpen not work for Redis storage

### ISSUE THREAD

**Issue by wenbinye** [ackintosh/ganesha#29]
Redis storage adapter does not implements saveLastFailureTime，and loadLastFailureTime suppose the key is `failureKey`, instead of `lastFailureKey`, so always got null value.

Change Ackintosh\Ganesha\Storage lastFailureKey as following works：

```php

    /**
     * @param  string $service
     * @return string
     */
    private function lastFailureKey($service)
    {
        return $this->supportRollingTimeWindow() 
            ? $this->failureKey($service) 
            : $this->prefix($service) . self::KEY_SUFFIX_LAST_FAILURE_TIME;
    }
```

**Comment by ackintosh** [comment]
@wenbinye Thanks for reporting the issue! I'll have a look this weekend. 😉

**Comment by ackintosh** [comment]
@wenbinye I've fixed the issue on #30. which is released as [v0.2.2](https://github.com/ackintosh/ganesha/releases/tag/0.2.2). Thank you again. 👍
